### PR TITLE
Modify ops guide for service releases

### PR DESCRIPTION
**Pull request by kwsutter**
Signed-off-by: Kevin Sutter <kwsutter@gmail.com>

Per the Spec Committee call today, we need to update the JESP and the Operations Guide to define the process for delivering a specification service release (x.y.z).

I am working with @ivargrimstad on this exercise.  I will leave this PR in Draft Mode until we are happy with the content.  And, then we'll communicate it to the wider Spec Committee for review before allowing this to be formally reviewed, approved, and merged.

### CODE REVIEW

**Review by ivargrimstad — APPROVED** [review]
LGTM

**Comment by waynebeaton** [comment]
Specification committee ballot approval of service releases is required per the EFSP.

**Comment by kwsutter** [comment]
> Specification committee ballot approval of service releases is required per the EFSP.

Thanks for pointing that out in PR #1050.  Let's have that discussion over there since the JESP decision will drive the Ops Guide.  Thanks!

File: content/committees/specification/operations/index.md [hunk 1]
```diff
@@ -355,6 +358,21 @@ The specification committee will promote the specification project to jakarta.ee
 Links: <https://github.com/jakartaee/jakarta.ee>,
 <https://gohugo.io/documentation/>
 
+### Creating a Final Service Release Specification # {#creating_a_final_service_release_specification}
+    
+There is no formal release review required for a Specification service release (x.y.z) as long as the JESP definition of a "service release" is adhered to.
+That is, no functionality changes are permitted in a service release.
```
Comment (starksm64) [inline content/committees/specification/operations/index.md]: That is, no functionality changes or increase of IP scope are permitted in a service release.

Comment (kwsutter) [reply content/committees/specification/operations/index.md]: Done.  Thanks!

**Comment by kwsutter** [comment]
With unanimous approval by the Steering and Specification Committees, this PR is ready to be merged.  Thank you!

> Discussion:
> Kevin presented the issue (described above) -- **Unanimous approval** by Spec. committee members.
> Would like to call a super-majority vote on a resolution to adopt JESP 1.3 proposal as drafted.
> Fujitsu  - yes
> IBM - yes
> Oracle - yes
> Payara - yes
> Red Hat - yes
> Tomitribe - yes
> Participant member representative - yes
> The resolution passed with **unanimous approval** of members present.

**Pull request merged**